README.md

```diff
@@ -1003,6 +1003,48 @@ const MyResponsiveDiv = glamorous.div({
 ```
 </details>
 
+In addition to all this, you can reference a `glamorous` component inside the
+styles of other `glamorous` components (this works because they each implement
+the `toString` method):
+
+<details>
+<summary>Contextual Overrides</summary>
+
+This API is inspired by `styled-components`.
+
+```javascript
+const Link = glamorous.a({
```

Comment:
Being able to use components (viz classes) as selectors is very useful, but only very occasionally. In general, good encapsulation/modularity means you don't want to use this pattern very often – but it's very useful in the small set of cases where it's useful as all.

Speaking for myself, if I were to build such an API, I would not want components to be usable as selectors by default; I want to increase (a bit) the friction of doing this, as a hint to users that this pattern is one that should be used when necessary, rather than casually.

Or another way to put it is that most componentized styled components probably shouldn't accept arbitrary style overrides by default.